myhpi/static/css/myHPI.css

```diff
@@ -205,19 +205,11 @@ h1.toc-title::after {
     height: 0px;
 }
 
-.print-shown {
+.d-print-block {
     display: none;
 }

```

Comment:
Out if curiosity: Why do we need this CSS here? Shouldn't `d-none d-print-block` ([see example here](https://getbootstrap.com/docs/5.2/utilities/display/#display-in-print))

Reply:
Because I didn't know